src/components/Loans/OpenLoans/components/ActionsDropdown/ActionsDropdown.js

```diff
@@ -77,6 +77,18 @@ class ActionsDropdown extends React.Component {
             <FormattedMessage id="stripes-smart-components.cddd.changeDueDate" />
           </Button>
         </IfPermission>
+        { get(loan, ['item', 'status', 'name']) !== 'Declared lost' &&
+          <Button
+            buttonStyle="dropdownItem"
+            data-test-dropdown-content-declare-lost-button
+            onClick={(e) => {
+              handleOptionsChange({ loan, action:'declareLost' }, e);
```

Comment:
```suggestion
              handleOptionsChange({ loan, action: 'declareLost' });
```
Event param can be ommited as it is not used.